.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: krew-index CI
+
+on: 
+  push:
+    branches: 
+      - '*'
+    tags: 
+      - 'v*.*.*'
+  pull_request:
+    branches: 
+      - '*'
+  release:
+    types: [published, created, edited]
+
+jobs:
+  build:
+    name: Build and Tags
+    runs-on: ubuntu-latest
+    env: 
+      KREW_VERSION: v0.3.3
+      GO111MODULE: on
+    steps:
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Install kubectl
+      run: |
+        sudo curl -fsSL -o /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.13.1/bin/linux/amd64/kubectl
+        sudo chmod +x /usr/bin/kubectl
+
+    - name: Install krew
+      run: |
+        cd "$(mktemp -d)" &&
+        curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/download/${KREW_VERSION}/krew.{tar.gz,yaml}" &&
+        tar zxvf krew.tar.gz &&
+        ./krew-linux_amd64 install --manifest=krew.yaml --archive=krew.tar.gz
+        
+    - name: Validate updated plugin manifests
+      run: git diff --name-only --diff-filter=AM $TRAVIS_BRANCH...HEAD |
```

Comment:
as you can imagine $TRAVIS_BRANCH won't work here.
please find the github equivalent.

@corneliusweig my this comment is also a good proof that without bash strict mode enabled, github scripts are powerless. they don't even notify us about undefined variables.

@RubenBaez please add `set -euo pipefail;` to the beginning of this script (as well as the script above).

Reply:
Aww cr*p. Good that you found this!

Reply:
Yes, good observation. I fixed it.